tests/components/FilterSearch.test.tsx

```diff
@@ -0,0 +1,295 @@
+import { FilterSearch } from '../../src/components/FilterSearch';
+import { FilterSearchResponse } from '@yext/answers-headless-react';
+import { render, screen } from '@testing-library/react';
+import * as useSynchronizedRequestFunctions from '../../src/hooks/useSynchronizedRequest';
+import userEvent from '@testing-library/user-event';
+import * as searchOperations from '../../src/utils/search-operations';
+import { mockAnswersActions } from '../__utils__/mocks';
+
+jest.mock('@yext/answers-headless-react');
+
+const mockedFilterSearchResponse: FilterSearchResponse = {
+  sections: [{
+    label: 'People',
+    results: [
+      { value: 'John Doe',
+        filter: { fieldId: null, matcher: null, value: 'test' } },
```

Comment:
In this test data can we use unique filters for each result? Instead of every single one having value 'test'. We should also add a matcher and fieldId to be more realistic.

Reply:
I'm ok not setting uuid to anything since that's currently unused

Reply:
added values to the filter fields to the old data, but I am unsure about whether the values I added for filter are representative of real data, but I tried to make assumptions based on what I could gather from looking around the code.  I'll leave this unresolved for now and come back to it.